openedx/core/djangoapps/course_groups/migrations/0006_cohort_membership_data_migrate.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        pass
```

Comment:
Note this is currently an empty placeholder, I'll be handcrafting some SQL before we merge this PR into master.